src/QMCHamiltonians/CoulombPBCAB.h

```diff
@@ -82,6 +90,13 @@ struct CoulombPBCAB: public QMCHamiltonianBase, public ForceBase
   std::vector<RadFunctorType*> Vat;
   ///Short-range potential for each species
   std::vector<RadFunctorType*> Vspec;
+  ///Short-range potential (r*V) and potential derivative d/dr(rV) derivative for each ion
```

Comment:
General comment: we should avoid pointers if they can be handled cleanly and then makeClone can be simplified or removed. Let us discuss in the next re-factorization WG.

Reply:
Agreed.